City-of-Pori/pori-kada
# PORI-191 go to online shop button

To test:

1. drush fra -y && drush cc all
2. Create attraction card and make sure there is field called "Online shop". Fill it with some web address.
3. Save the attraction card and make sure on desktop view you are able to see two links pointing to the web address you just wrote there. One next to the title and one under the contact information. On mobile there should be only one under the contact information visible.

Notice: I also removed the normalize styles from visitpori theme since they are already added in the kada theme and were messing stuff up. Check the styles in general that the layout looks nice.
